docs/user-guide/controller/extensions.md

```diff
@@ -4,7 +4,7 @@ template: main.html
 
 # Iter8 controller extensions
 
-Iter8 can be easily extended to watch any type of Kubernetes object, including objects with custom resource definitions (CRDs) as part of a application version. 
+The Iter8 controller watches the Kubernetes resources that make up application versions as specified in a [routemap](../routemap.md). When the controller is installed, Kubernetes must be configured to give Iter8 permission to watch the desired resource types. The set of resource types (including those with a custom resource definition (CRD)) Iter8 is specified in a `resourceTypes` map in the [`values.yaml`](https://github.com/iter8-tools/iter8/blob/v0.18.3/charts/controller/values.yaml) file used to configure the controller. Iter8 can be easily extended to watch additional objects by adding to this map. 
```

Comment:
```suggestion
The Iter8 controller watches the Kubernetes resources that make up application versions as specified in a [routemap](../routemap.md). When the controller is installed, Kubernetes must be configured to give Iter8 permission to watch the desired resource types. The set of resource types (including those with a custom resource definition (CRD)) is specified in a `resourceTypes` map in the [`values.yaml`](https://github.com/iter8-tools/iter8/blob/v0.18.3/charts/controller/values.yaml) file used to configure the controller. Iter8 can be easily extended to watch additional objects by appending to this map. 
```

Reply:
Nothing wrong with adding, just think appending sounds nice and adds some variety